src/classes/Language.as

```diff
@@ -4,6 +4,7 @@
 
 package classes
 {
+    import classes.ui.Text;
```

Comment:
Unnecessary?

Reply:
The function `font()` contains this statement:

`return Text.isUnicode(testStr) ? UNI_FONT_NAME : FONT_NAME;`

For this function to work, we need to import the `Text` class from `classes.ui`.

Reply:
Sounds good.